edge-modules/SimulatedTemperatureSensor/src/Program.cs

```diff
@@ -126,10 +127,24 @@ static Task<MessageResponse> ControlMessageHandle(Message message, object userCo
                     }
                     else
                     {
-                        //NoOp
+                        // NoOp
                     }
                 }
             }
+            catch (JsonSerializationException)
+            {
+                var messageBody = JsonConvert.DeserializeObject<ControlCommand>(messageString);
```

Comment:
Curious what scenario this supports.

Reply:
This is for tutorial with ASA module, we recently updated the event output format to single JSON event (or multiple JSON events delimited by ",").

Reply:
Is the scenario with the array of messages still valid?

Reply:
not being used by current ASA module. keep it for backward compatible